README.md

```diff
@@ -3,29 +3,69 @@ Scripts for easy backup and restore of Docker volumes
 
 ## Usage
 
-```
+```bash
 ./docker_volume_backup.sh {compose_file_path} {project_name} {backup_path} {backup_or_restore} {restore_date}
+# or
+./docker_full_backup.sh {compose_file_path} {project_name} {backup_path} {backup_or_restore} {restore_date}
 ```
 
 ## Examples
 
+### docker_volume_backup.sh
+
 Backup
 
-```
+```bash
 ./docker_volume_backup.sh /home/kiview/Gitlab/docker-compose.yml gitlab $(pwd)/backup backup
 ```
 
 Restore
 
-```
+```bash
 ./docker_volume_backup.sh /home/kiview/Gitlab/docker-compose.yml gitlab $(pwd)/backup restore 2016-10-19
 ```
+
+### docker_full_backup.sh
+
+```bash
+MODE=backup # or restore
+
+path=/opt/mydocker/project1/docker-compose.yml
+project=project1
+BACKUP_DIR=/tmp/mybackups
+DATE=xxxx-xx-xx
+
+#export EXCLUDE_CONTAINER=1
+#export EXCLUDE_VOLUMES=1
+
+./docker_full_backup.sh $path $project $BACKUP_DIR $MODE $DATE
+```
+
 ## Docker Container Usage
-After building your container, you can use it like this:
+
+After building your container,
+
+```bash
+docker build -t docker_volume_backup .
+# or
+./build.sh
 ```
-docker run -v "/home/kiview/Gitlab/:/project" \
-    -v "$(pwd)/backup:/backup" \
-    -v /var/run/docker.sock:/var/run/docker.sock \
-    docker_volume_backup:latest Gitlab /backup backup
+
+you can use it like this:
+
+```bash
+PROJECT_DIR   # path to directory that contains the docker files, e.g. docker-compose.yml, Dockerfile, ...
```

Comment:
Same as above, this will only work if the example is in a bash file. The old examples could be run directly from the command line.